backend/modules/sse/sse_consumer_impl.py

```
@@ -1,74 +1,78 @@
 import asyncio
-import logging
-import uuid
+from typing import TYPE_CHECKING, override
```

Pylint (3.3.7)

[error] No name 'override' in module 'typing'

(E0611)